packages/server/modules/core/graph/resolvers/commits.js

```diff
@@ -216,12 +215,7 @@ module.exports = {
     },
 
     async commitReceive(parent, args, context) {
-      // if stream is private, check if the user has access to it
-      const stream = await getStream({ streamId: args.input.streamId })
-
-      if (!stream.public) {
-        await authorizeResolver(context.userId, args.input.streamId, 'stream:reviewer')
-      }
+      await authorizeResolver(context.userId, args.input.streamId, 'stream:reviewer')
```

Comment:
The stream.public check is redundant now that authorizeResolver properly checks for isPublic